tests/frameworks/layers/l2l4_network_creation_test.py

```diff
@@ -834,23 +830,23 @@ def generatePattern(self, max, size):
     return cellsIndices[:size]
 
 
-  def getL4PredictiveCells(self, column):
-    """Returns the predictive cells in L4."""
-    return set(column._tm.getPredictiveCells())
+  def getL4PredictedCells(self, column):
+    """Returns the predicted cells in L4."""
```

Comment:
Could you specify clearly what the term "predictedCells" means? e.g. are these the cells that have active segments as a result of the last call to compute?